IbisLib/pointerobject.h

```diff
@@ -88,6 +92,13 @@ private slots:
     double m_pointerUpDir[3];
     double m_tipLength;
 
+    int m_calibrating;
+    vtkCriticalSection * m_calibrationMutex;
```

Comment:
I think this is not needed anymore. It was to control access to the calibration data from multiple thread. PointerObject doesn't use multiple threads.

Reply:
Fixed in the latest commit.

Reply:
I'm removed the mutex. The m_calibrating var is still used in the update function though.